src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/controller/ReportsControllerTest.java

```diff
@@ -63,27 +69,47 @@
 
     @Test
     public void should_return_result_generated_by_the_service() throws Exception {
-        final EnvelopeCountSummary countSummary = new EnvelopeCountSummary(
-            100, 11, "hello", LocalDate.of(2019, 1, 14)
-        );
 
-        given(reportsService.getCountFor(countSummary.date, false))
-            .willReturn(singletonList(countSummary));
+        final EnvelopeCountSummary countSummaryOne = new EnvelopeCountSummary(
+            150, 10, "container1", LocalDate.of(2021, 3, 4)
+        );
+        final EnvelopeCountSummary countSummaryTwo = new EnvelopeCountSummary(
+            150, 10, "container2", LocalDate.of(2021, 3, 4)
```

Comment:
I meant they do not need to be same numbers, both has 150 and 10.

Reply:
changed